### PR TITLE
Build anbox-cloud landing page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build new Anbox cloud landing page
- Some links are missing as these pages are yet to be built: `Learn more about virtual devices`, `Learn more about secure apps`, `Learn more about game streaming`, `Learn more about automated testing`
- Blocked: Missing link `Deploy on AWS` - waiting for response from Galem.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the page against the [copy doc](https://docs.google.com/document/d/1SfpOBeDJUZj2ZYq4V8lHOUbQF9Yzo7mp5NajJDVwFMI/edit#heading=h.pnlo30ndk43n) 

## Issue / Card

Fixes [#4006](https://github.com/canonical-web-and-design/web-squad/issues/4006)

### CODE REVIEW

**Comment by anasereijo** [comment]
- [ ] move 'in the' to the 2nd line
Screenshot 2021-05-05 at 16 22 51

- [ ] titles should be `<h3>`
Screenshot 2021-05-05 at 16 25 03

- [ ] Link missing from 'Stream games to millions'
Screenshot 2021-05-05 at 16 26 44

- [ ] Link missing from 'Virtualise phones and tablets'
Screenshot 2021-05-05 at 16 28 24

- [ ] Link missing from 'Secure mobile apps'
Screenshot 2021-05-05 at 16 31 15

- [ ] Link missing from 'Automate app testing at scale'
Screenshot 2021-05-05 at 16 32 01

**Learn more about Android in the cloud**
- [ ] title should be `<h3>`

update icons with correct links:
- [ ] [webinar](https://assets.ubuntu.com/v1/816ae23b-Webinar+-+white.svg)
- [ ] [whitepaper](https://assets.ubuntu.com/v1/4ab8ff35-Whitepaper+-+white.svg)
- [ ] [blog](https://assets.ubuntu.com/v1/271921af-Datasheet+-+white.svg)

**Quotes section:**
- [ ] please use the small option. go to [this page](https://vanillaframework.io/docs/examples/patterns/pull-quotes/variants) and find 'Small variant - image, 1 paragraph, with cite'
- [ ] I would also like to change the logo sizes but I can't do it when I use the 'inspect element'. can we go through this together?
- [ ] typo:
-
Screenshot 2021-05-05 at 16 57 15


**CLOUDS SERVICE PROVIDERS**
- [ ] the title styling doesn't look right to me?
Screenshot 2021-05-05 at 16 55 49
- [ ] can we change the logo sizes or is this automatic?

**Comment by bethcollins92** [comment]
Hi @anasereijo the logo section **title** is new I think - I've used this class `p-logo-section__title` from [here.](https://vanillaframework.io/docs/patterns/logo-section) Also the size of the logos are set in Vanilla to a maximum height which is why the wide logos look a lot bigger - I think this is like this so the quotes all lines up correctly? Happy to jump on a call and look at any of this with you though

**Comment by anasereijo** [comment]
@bethcollins92 thanks for making all those changes. this is looking good +1 but let me see it again when you have the video

**Comment by bethcollins92** [comment]
@anasereijo Added the video and removed the 6th matrix item

**Comment by anasereijo** [comment]
@bethcollins92 this is looking good just some tiny things (hopefully)

- [ ] add the 6th whitepaper

- [ ] on small screens put 1 item per row

Screenshot 2021-05-17 at 15 30 25

- [ ] small screens: can we get rid of the card and just have the buttons without the titles? also, seems like both of the contact us buttons takes us to the same page, so on small screens, we can just have one?
Screenshot 2021-05-17 at 15 31 27

**Comment by bethcollins92** [comment]
@anasereijo demo is updated and ready for another design review please

**Comment by anasereijo** [comment]
LGTM thanks

File: static/sass/_pattern_deployment-table.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@mixin anbox-p-deployment-table {
+  .table-cell {
+    @media (min-width: $breakpoint-medium) {
+      justify-content: center !important;
+      text-align: center !important;
+    }
```
Comment (ClementChaumel) [inline static/sass/_pattern_deployment-table.scss:6]: justify-content doesn't seem to do anything and the text-align seem to work without the `!important`
image
So if it is no crucial I'd rather that we removed `!important`

File: static/sass/_pattern_strip.scss [hunk 2]
```diff
@@ -1,18 +1,201 @@
 // sass-lint:disable no-color-literals hex-notation
 @mixin anbox-p-strip {
+  @include anbox-p-strip-suru-accent;
   @include anbox-p-strip-suru-topped;
+  @include anbox-p-strip-suru-image;
+}
+
+@mixin anbox-p-strip-suru-accent {
```
Comment (ClementChaumel) [inline static/sass/_pattern_strip.scss:8]: Is the anbox suru different from vanilla one?

**Review by ClementChaumel — APPROVED** [review]
Looking good!

File: static/sass/_pattern_deployment-table.scss (hunk 1 above)
Comment (bethcollins92) [reply static/sass/_pattern_deployment-table.scss]: Fixed - thanks!

File: static/sass/_pattern_strip.scss (hunk 2 above)
Comment (bethcollins92) [reply static/sass/_pattern_strip.scss:8]: I'm not sure how to import surus from vanilla? the class didn't work unless I added the styles directly. Happy to update this though if I'm missing something

Comment (bethcollins92) [reply static/sass/_pattern_strip.scss]: Removed the `p-strip--suru-accent` scss and added the colour variables - thanks! (and removed `p-strip--suru-topped` as this isn't used anymore)

**Comment by bethcollins92** [comment]
@SirSamTumless it's just that "short link" missing now - everything else is ready to go. I'm thinking I'll just remove the button and whenever the link is ready we can add it back? Let me know what you think

File: templates/index.html [hunk 3]
```diff
@@ -7,137 +7,358 @@
...
+      <p class="p-heading--4 u-sv3">Unique cloud virtualisation technology, boundless possibilities for Android</p>
+    </div>
+  </div>
+  <div class="row u-vertically-center">
+    <div class="col-6">
+      <h3>Stream games to millions</h3>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Better unit economics thanks to high container density</li>
+        <li class="p-list__item is-ticked">Low latency and high frame rates</li>
+        <li class="p-list__item is-ticked">ASupport for x86 and Arm</li>
```
Comment (morphis) [inline templates/index.html:76]: `ASupport` -> `Support`

File: templates/index.html [hunk 4]
```diff
@@ -7,137 +7,358 @@
...
+                url="https://assets.ubuntu.com/v1/4ab8ff35-Whitepaper+-+white.svg",
+                alt="",
+                height="32",
+                width="32",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
+              ) | safe
+            }}
+            <p class="p-heading-icon__title p-heading--4" style="line-height: 1.5rem; color: #fff">Whitepaper</p>
```
Comment (morphis) [inline templates/index.html:270]: The link is a blog post rather than a whitepaper

File: templates/index.html [hunk 5]
```diff
@@ -149,190 +370,211 @@ <h2>From our partners</h2>
           ) | safe
         }}
         <p class="p-pull-quote__quote">The combination of Ampere’s Arm-based servers with a provisioned virtualisation solution like Canonical’s Anbox Cloud delivers the flexible, high-performance and secure infrastructure that developers need in order to deliver a better user experience for consumers.</p>
-        <cite class="p-pull-quote__citation">Jeff Wittich<br>SVP of Products at <a href="https://amperecomputing.com/" class="p-link--external">Ampere</a></cite>
+        <cite class="p-pull-quote__citation">Jeff Wittich, SVP of Products at <a class="p-link--external" href="https://amperecomputing.com/">Ampere</a></cite>
```
Comment (morphis) [inline templates/index.html:373]: Any reason why we link Ampere Computing but not the others giving us a quote?

**Review by morphis — COMMENTED** [review]
This is looking great! Thanks a lot for the hard work.

Just found a couple of small things.

File: templates/index.html (hunk 5 above)
Comment (bethcollins92) [reply templates/index.html:373]: @morphis Not sure I was just following the copy doc? I've made the other suggested changes, thanks. I'm going to remove the Deploy on AWS button for now as we don't have the link.

Comment (bethcollins92) [reply templates/index.html:373]: Do you want to add links to the others giving us a quote?

**Pull request merged**